OrbitGl/TypesDataView.cpp

```diff
@@ -223,10 +223,10 @@ void TypesDataView::OnSort(int a_Column,
   auto MemberID = static_cast<Type::MemberID>(s_HeaderMap[a_Column]);
```

Comment:
For the first column we get Type::INDEX, which is some constant (say 10) we they use this constant as an index to array of size of number of columns... In this particular case columns are a subset of some possible Types and the size is 10, which leads to out of bounds access for vector of size 10.

Seems like it's a bug